PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,113 @@
 ## Brief description
 
-...
+#### Top Nav element for PaperCSS
```

Comment:
why has this file been edited?

Reply:
Sorry, it is my first time contributing to an open source via github. I will fix it